clang/lib/StaticAnalyzer/Checkers/GenericTaintChecker.cpp

```diff
@@ -733,13 +726,23 @@ void GenericTaintChecker::initTaintRules(CheckerContext &C) const {
       {{CDF_MaybeBuiltin, {{"stpcpy"}}},
        TR::Prop({{1}}, {{0, ReturnValueIndex}})},
       {{CDF_MaybeBuiltin, {{"strcat"}}},
-       TR::Prop({{1}}, {{0, ReturnValueIndex}})},
+       TR::Prop({{0,1}}, {{0, ReturnValueIndex}})},
```

Comment:
```suggestion
       TR::Prop({{0, 1}}, {{0, ReturnValueIndex}})},
```
Add space after comma here and in the test code.